index.html

```diff
@@ -0,0 +1,857 @@
+                        </h5>
+
+                        </div>
+
+                    </div>
+                    <div class="picture-sponsored">
+                        <div class="sponsor-elements-wrapper">
+                            <img src="https://d.newsweek.com/en/full/1556493/ohio-northern-university-pettit-college-law.jpg?w=343&h=229&f=83cc4a2b96cbd472e102cf04f99fc5be" alt="">
+                            <div class="picture-sponsor-title-summary-wrapper">
+                                <h3><a href="">Have You Thought About A Career In Law? If So, Where Should You Get Qualified?</a></h3>
+                                <p>
+                                    The rewards for a career in the legal profession are many and varied, and demand for lawyers is high. So which schools produce the best lawyers?
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="picture-sponsored">
+                        <div class="sponsor-elements-wrapper">
+                            <img src="https://d.newsweek.com/en/full/1551978/media-school-indiana-university.jpg?w=343&h=229&f=2acd7418079f8cde5545fea5bb3815bd" alt="">
+                            <div class="picture-sponsor-title-summary-wrapper">
+                                <h3><a href="">Good Journalists Are More Important Than Ever in 2020</a></h3>
+                                <p>
+                                    The world desperately needs sharp, incisive and capable journalists to cut through the noise with measured and nuanced takes on today's pressing issues. For these looking to find their voices, an education at a leading journalism school is invaluable.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="picture-sponsored">
+                        <div class="sponsor-elements-wrapper">
+                            <img src="https://d.newsweek.com/en/full/1550618/americas-most-responsible-companies-2020.jpg?w=343&h=229&f=228a31483952835462a9e24494fc24f7" alt="">
+                            <div class="picture-sponsor-title-summary-wrapper">
+                                <h3><a href="">Socially Responsible Companies</a></h3>
+                                <p>
+                                    Spotlight 300 companies that stand up as good citizens and good neighbors.
+                                </p>
+                            </div>
+                        </div>
+
+                    </div>
+                </div>
+
+            </div>
+            
+            
+        </article>
+        <article class="bottom-of-page container-fluid">
+            <div class="row">
+                <div class="col-12 bottom-of-page-title">
```

Comment:
Let us add ```padding``` to ```<div class="bottom-of-page-title">```

Reply:
I changed the property from "none" to "thin" to display that the property is working very well despite the CSS validator error.